src/tests/ftest/launch.py

```diff
@@ -2149,10 +2291,12 @@ def main():
 
     # Setup the user environment
     set_test_environment(args)
-
+    vmd_flag = False
     # Auto-detect nvme test yaml replacement values if requested
-    if args.nvme and args.nvme.startswith("auto"):
-        args.nvme = get_nvme_replacement(args)
+    if args.nvme and args.nvme.startswith("auto_vmd"):
+        args.nvme, vmd_flag = get_vmd_replacement(args)
+    elif args.nvme and args.nvme.startswith("auto"):
+        args.nvme, vmd_flag = get_nvme_replacement(args)
```

Comment:
Does this method really need to return a flag that has a fixed value and has the same default value set on line 2294?

Reply:
This comment was only supposed to apply to 2299.

Reply:
Yes, I can remove it from the get_nvme_replacement function.